iothub/service/src/Jobs/ScheduledDeviceMethod.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Contains fields used when creating a job to run a device method on one or multiple devices.
+    /// </summary>
+    public class ScheduledDeviceMethod
```

Comment:
```suggestion
    public class ScheduledDirectMethod
```

since "Direct Method" is the concept's name, not "Device Method"